rules/redis.yml

```diff
@@ -0,0 +1,45 @@
+groups:
+  - name: Redis
+    rules:
+      - alert: Redis Down
+        expr: redis_up == 0
+        for: 5m
+        labels:
+          severity: error
+        annotations:
+          summary: "Redis down (instance {{ $labels.instance }})"
+      - alert: Redis Missing Backup
+        expr: time() - redis_rdb_last_save_timestamp_seconds > 60 * 60 * 24
+        for: 5m
+        labels:
+          severity: error
+        annotations:
+          summary: "Missing backup for 24 hours (instance {{ $labels.instance }})"
+      - alert: Redis Replication Broken
+        expr: delta(redis_connected_slaves[1d]) < 0
+        for: 5m
+        labels:
+          severity: error
+        annotations:
+          summary: "Replication broken (instance {{ $labels.instance }})"
+      - alert: Redis Too Many Connections
+        expr: redis_connected_clients > 100
+        for: 5m
+        labels:
+          severity: warning
+        annotations:
+          summary: "Too many connections (> 100) (instance {{ $labels.instance }})"
+      - alert: Redis Not Enough Connections
+        expr: redis_connected_clients < 5
```

Comment:
is this is good number?

Reply:
I still don't know. Please suggest me, or tell me how to get a good number for this.

Reply:
for now we can go with this number